src/components/legend.ts

```diff
@@ -63,6 +64,25 @@ export module Components {
     }
 
     /**
+     * Gets the Formatter for the text.
+     */
+    public formatter(): Formatter;
+    /**
+     * Sets the Formatter for the text.
+     *
+     * @param {Formatter} formatter
+     * @returns {Legend} The calling Legend.
+     */
+    public formatter(formatter: Formatter): Legend;
+    public formatter(formatter?: Formatter): any {
+      if (formatter === null) {
```

Comment:
This seems incorrect.  Now the getter can only be called with `legend.formatter(null)` it seems.